packager/util/config.go

```diff
@@ -185,7 +187,11 @@ func ReadConfig(configBytes []byte) (*Config, error) {
 	// TODO(twifkak): Return an error if the TOML includes any fields that aren't part of the Config struct.
 
 	if config.Port == 0 {
-		config.Port = 8080
+		if i, err := strconv.ParseInt(os.Getenv("PORT"), 10, 32); err == nil {
```

Comment:
1. Use [strconv.Atoi](https://golang.org/pkg/strconv/#Atoi) for a bit-depth that matches `int` so there won't be overflow.
2. If i <= 0, return an error.